libs/components/src/next/form/radio/stories/vl-radio.stories-arg.ts

```diff
@@ -0,0 +1,116 @@
+import { CATEGORIES, defaultArgs, defaultArgTypes, TYPES } from '@domg-wc/common-storybook';
+import { ArgTypes } from '@storybook/web-components';
+import { RadioDefaults } from '../index';
+import { formControlArgs } from '../../form-control/stories/form-control.stories-arg';
+import { action } from '@storybook/addon-actions';
+import { inputFieldArgs } from '@domg-wc/components/next/form/input-field/stories/vl-input-field.stories-arg';
+
+export const radioArgs: typeof defaultArgs & typeof RadioDefaults & { onVlChecked: () => void; contentSlot: string } = {
+    ...defaultArgs,
+    ...RadioDefaults,
+    contentSlot: '',
+    onVlChecked: action('vl-checked'),
+};
+
+export const radioArgTypes: ArgTypes<typeof radioArgs> = {
+    ...defaultArgTypes(true),
+    value: {
+        name: 'value',
+        description: 'De value van de radio.',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: radioArgs.value },
+        },
+    },
+    block: {
+        name: 'block',
+        description: 'Duidt aan dat de component de volledige breedte van zijn parent mag innemen.',
+        table: {
+            type: { summary: TYPES.BOOLEAN },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: radioArgs.block },
+        },
+    },
+    readonly: {
+        name: 'readonly',
+        description: 'Duidt aan dat het veld enkel leesbaar is.',
+        table: {
+            type: { summary: TYPES.BOOLEAN },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: inputFieldArgs.readonly },
+        },
+    },
+    id: {
+        name: 'id',
+        description: 'Het id van het veld.',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: formControlArgs.id },
+        },
+    },
+    name: {
+        name: 'name',
+        description: 'De naam van het veld.',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: formControlArgs.name },
+        },
+    },
+    checked: {
+        name: 'checked',
+        description: 'Vinkt de radio aan of uit.',
+        table: {
+            type: { summary: TYPES.BOOLEAN },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: radioArgs.checked },
+        },
+    },
+    contentSlot: {
```

Comment:
Kan je het slot beneden in de args zetten? Net boven de events.

Reply:
Done.